docs/_data/menu-en.yml

```
@@ -153,6 +153,8 @@
               url: /en/api/S3-API.html
     - title: Glossary
       url: /en/reference/Glossary.html
+    - title: Membership Management
+      url: en/reference/Alluxio-Membership-Management.html
```

```suggestion
      url: /en/operation/Alluxio-Membership-Management.html
```
- need to start url with `/`
- i think `operation` is more suitable compared to `reference` since it is more operational guidance as opposed to a long list of strings. @ccmao1130 any opinions about which category?

---

agree with operation!

---

ok i will move to operation folder